src/temp_sensor.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(esp32c3)]
+use esp_idf_sys::{
```

Comment:
when you import a crate its not automatically scoped to all the child modules inside the file

Reply:
https://github.com/esp-rs/esp-idf-hal/pull/337#discussion_r1391807390

Reply:
Resolved by https://github.com/esp-rs/esp-idf-hal/pull/337/commits/01ce494260da154db5794c89a94ec77bb9bdef14.